extensions/src/libadalang-helpers.ads

```diff
@@ -302,6 +309,13 @@ package Libadalang.Helpers is
            (Parser, Long => "--symbolic-traceback",
             Help         => "Show symbolic tracebacks for exceptions");
 
+         package Exit_On_Missing_File is new Parse_Flag
+           (Parser,
+            Short => "-é", Long => "--éhéhé",
+            Help  => "Behavior when encountering missing files. By default,"
+            & "abort, but if this is passed, will continue and present results"
```

Comment:
Missing spaces (I suggest to add them before “abort” and “anyway” so that they are visible from the first look)